packages/@aws-cdk/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -345,25 +352,6 @@ export class Nodegroup extends Resource implements INodegroup {
       tags: props.tags,
     });
 
-    if (props.launchTemplateSpec) {
-      if (props.diskSize) {
-        // see - https://docs.aws.amazon.com/eks/latest/userguide/launch-templates.html
-        // and https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-eks-nodegroup.html#cfn-eks-nodegroup-disksize
-        throw new Error('diskSize must be specified within the launch template');
-      }
```

Comment:
What about this validation? Why is it ok to drop it now?

Reply:
it's being move to here because that's all the other props validation happens:
https://github.com/aws/aws-cdk/blob/19083733fc007a78852b7654c6ab23831f35c641/packages/%40aws-cdk/aws-eks/lib/managed-nodegroup.ts#L283-L287

Reply:
Apologies, I missed that. Looks good.